kyo-core/shared/src/main/scala/kyo/Channel.scala

```diff
@@ -78,6 +80,24 @@ object Channel:
                 }
             }
 
+        /** Takes [[n]] elements from the channel, asynchronously blocking if necessary.
```

Comment:
If you could add a warning that the computation will wait while the channel doesn't provide all values?